src/oc_erchef/apps/chef_index/src/chef_index_http.erl

```diff
@@ -67,11 +67,11 @@ post(Url, Body, Headers) when is_list(Body) ->
 post(Url, Body, Headers) ->
     request_with_caught_errors(Url, post, Body, Headers).
 
--spec delete(list(), iolist() | binary()) -> ok | {error, term()}.
+-spec delete(iolist(), iolist() | binary()) -> ok | {error, term()}.
```

Comment:
As discusssed, we could potentially change all the specs in this file, but we want the minimal change to get this building as the actual specs are very suspect at the moment and extensive changes may be needed.